DigitalOffice/Services/ProjectService/Models/EditProjectRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ProjectService.Models
+{
+    public class EditProjectRequest
+    {
```

Comment:
Why request does not contain ProjectId?

Reply:
Because we sent it via query and route?
Anyway, you want to assign Guid from query to the model?

Reply:
What is the problem with sending id in body?

Reply:
Sounds like a plan. I'll make changes.